Library/Formula/fish-tank.rb

```diff
@@ -0,0 +1,15 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/joelthelion/autojump.git"
+
+  def install
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```

Comment:
If `fish-tank` runs tests in `fish`, you should have `fish-tank` run a test you expect to pass in the formula's test.

This formula should `depends_on "fish"` if it is not useful without fish.